### PR TITLE
ROU-4763: Add ability to to GoogleMaps to add localization by setting the language and region

### PR DESCRIPTION
**Pull request by bmarcelino-fe**
This PR is for adding the **Localization** feature to `GoogleMaps`, `StaticMap` and `SearchPlaces`  by setting the **language** and **region** through `OptionalConfigs` on each component.

### What was happening
* The user to apply a localization on a google maps, he need to add it on APIKey value as workaround. With this, the google maps will have the language and region applied.

### What was done
* Add the Localization parameter into OptionalConfigs on GoogleMaps, StaticMap and SearchPlaces
* Review the TypeScript to add the localization (language and region) into consideration on loaded script by google
image
image

### Test Steps
1. Open each Sample Page
2. **Expected:** Map will rendered in Japan language _(On StaticMap, you can change the language and toogle the map visibility to apply the changes.)_

### Checklist
* [X] tested locally
* [X] documented the code
* [X] clean all warnings and errors of eslint
* [ ] requires changes in OutSystems (if so, provide a module with changes)
* [ ] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

File: src/Providers/Maps/Google/SharedComponents/MapsAndSearchPlaces.ts [hunk 1]
```diff
@@ -42,6 +50,8 @@ namespace Provider.Maps.Google.SharedComponents {
 			// the callback will only work of 1st map on the page
 			googleMapsLoadPromise.then(cb);
 		}
+
+		console.log(this.config);
```
Comment (rugoncalves) [inline src/Providers/Maps/Google/SharedComponents/MapsAndSearchPlaces.ts:54]: Remove the console log! :)

**Review by rugoncalves — DISMISSED** [review]
Leaflet doesn't support localization?

File: src/Providers/Maps/Google/SharedComponents/MapsAndSearchPlaces.ts (hunk 1 above)
Comment (bmarcelino-fe) [reply src/Providers/Maps/Google/SharedComponents/MapsAndSearchPlaces.ts:54]: Done

**Comment by bmarcelino-fe** [comment]
> Member

**Comment by bmarcelino-fe** [comment]
> Leaflet doesn't support localization?

@rugoncalves The leaflet doesn't have support for localization, this only on google maps.

**Review by joselrio — APPROVED** [review]

**Pull request merged**